src/rgw/rgw_quota.cc

```diff
@@ -196,7 +165,9 @@ void RGWQuotaCache<T>::set_stats(const rgw_user& user, const rgw_bucket& bucket,
 }
 
 template<class T>
-int RGWQuotaCache<T>::get_stats(const rgw_user& user, const rgw_bucket& bucket, RGWStorageStats& stats, RGWQuotaInfo& quota, optional_yield y, const DoutPrefixProvider *dpp) {
+int
```

Comment:
I think you added new line by mistake, other than that LGTM

Reply:
Reformatted.

Reply:
Can you please squash the commits into a single commit?

Reply:
Done